added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/zero-to-hero/twitter-dapp/ProjectSetup.md

displayed_sidebar: developSidebar
---

<head>

>   <title>Build Decentralised Twitter in 50 minutes using Subsocial | Zero-to-Hero Tutorial</title>

Reviewer: ```suggestion
  <title>Build A Decentralized Twitter In 50 Minutes With Subsocial</title>
```

</head>

> The **Zero to Hero tutorial** is your chance to build a fully decentralized twitter app in 50 minutes using the Subsocial Starter.

Reviewer: ```suggestion
The **Zero to Hero tutorial** is your chance to build a fully decentralized Twitter app in 50 minutes using the Subsocial Starter.
```
